rancher-config/docker-compose.yml

```diff
@@ -26,5 +26,5 @@ services:
         splunk-sourcetype: docker
         splunk-source: pharmacy-finder
```

Comment:
I was just thinking about that. Feel like we should maybe rename the repo instead to `pharmacy-finder`?